client/packages/inventory/src/Stocktake/ListView/createStocktake.ts

```diff
@@ -0,0 +1,60 @@
+import { useFormatDateTime, useTranslation } from '@common/intl';
```

Comment:
Would remove this and refactor just the create function right now to use the new pattern. Have shared params with the modal as commented above